### PR TITLE
Some improvements from `more-streams`

### PR DESCRIPTION
**Pull request by rodrigok**
@RocketChat/core

### CODE REVIEW

File: packages/rocketchat-ui-message/message/popup/messagePopup.coffee [hunk 1]
```diff
@@ -160,7 +160,7 @@ Template.messagePopup.onCreated ->
 
 	template.records = new ReactiveVar []
 	Tracker.autorun ->
-		if template.data.collection.find?
+		if template.data.collection.findOne?
 			template.data.collection.find().count()
```
Comment (marceloschmidt) [inline packages/rocketchat-ui-message/message/popup/messagePopup.coffee]: Should you use findOne here?

Comment (marceloschmidt) [inline packages/rocketchat-ui-message/message/popup/messagePopup.coffee]: Also, it looks like it is not doing anything, but I may be wrong for not seeing the whole picture

Comment (rodrigok) [inline packages/rocketchat-ui-message/message/popup/messagePopup.coffee]: @marceloschmidt Yes, cuz array.find is valid too, see here https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Comment (marceloschmidt) [inline packages/rocketchat-ui-message/message/popup/messagePopup.coffee]: Ok, so in fact you're only checking if that's an array or a mongo collection? 
It looked strange that you check for `findOne` but then use `find`, that's all. And what does it do `template.data.collection.find().count()`?

Comment (rodrigok) [inline packages/rocketchat-ui-message/message/popup/messagePopup.coffee]: @marceloschmidt Yes, the count keep the code reactive to list changes

Comment (marceloschmidt) [inline packages/rocketchat-ui-message/message/popup/messagePopup.coffee]: Cool!

**Pull request merged**